index.html

```diff
@@ -160,13 +200,139 @@
+            background: #FFF;
+            display: flex;
+            padding: 35px 40px;
+            flex-direction: column;
+            align-items: flex-start;
+            gap: 15px;
+        }
+        .cardname {
+            color: var(--text-color, #252B42);
+            font-size: 16px;
+            font-style: normal;
+            font-weight: 700;
+            line-height: 24px; 
+            letter-spacing: 0.1px;
+        }
+        .cardheading {
+            display: flex;
+            align-items: center;
+            gap: 15px;
+        }
+        #fox {
+            color: var(--second-text-color, #737373);
+            font-size: 14px;
+            font-style: normal;
+            font-weight: 500;
+            line-height: 20px; 
+            letter-spacing: 0.2px;
+        }
+    </style>
+    <section class="why-us">
+        <div class="container">
+            <div class="why_us_content">
+            <h2 id="whyus">WHY CHOOSE US</h2>
+            <p id="newtonian">Problems trying to resolve the conflict between the two major realms of Classical physics:
+                Newtonian mechanics</p>
+            </div>
+            <div class="grid-container">
+                <div class="grid-item">
+                    <div class="cardheading">
+                    <img src="Project materials\bag .icn-lg.svg" alt="Bag">
+                    <h5 class="cardname">Digital Marketing</h5> 
+                    </div>
+                    <p id="fox"> the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="grid-item">
+                    <div class="cardheading">
+                    <img src="Project materials\market .icn-lg.svg" alt="Market">
+                    <h5 class="cardname">Digital Marketing</h5> 
+                    </div>
+                    <p id="fox"> the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="grid-item">
+                    <div class="cardheading">
+                    <img src="Project materials\note book .icn-lg.svg" alt="Notebook">
+                    <h5 class="cardname">Business Growing</h5>
+                    </div>
+                    <p id="fox"> the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="grid-item">
+                    <div class="cardheading">
+                    <img src="Project materials\people .icn-lg.svg" alt="People">
+                    <h5 class="cardname">National top 50 firms</h5>
+                    </div>
+                    <p id="fox"> the quick fox jumps over the lazy dog the quick fox jumps over the lazy dog</p>
+                </div>
+                <div class="grid-item">
+                    <div class="cardheading">
+                    <img src="Project materials\diagram .icn-lg.svg" alt="Diagram">
```

Comment:
" .icn-lg" inside the icon name is redundant. And no spaces should be in file names. Please cleanup the naming